### ISSUE TITLE
Facial landmark detection

### ISSUE THREAD

**Issue by blackshrub** [eduardolundgren/tracking.js#223]
Is tracking.js already capable to detect facial landmark?
How can we specify a particular face feature with FAST method in tracking.js?

**Comment by revolunet** [comment]
hi, any progress on this ?

**Comment by blackshrub** [comment]
@revolunet Nope, I'm finally using another js lib that has supported it out of the box.

**Comment by alexiskattan** [comment]
@blackshrub  which one?

**Comment by blackshrub** [comment]
@alexiskattan https://github.com/Tastenkunst

**Comment by Jylomaki** [comment]
Now there is also CLMTrack.js to do facial landmarking. Seems like it uses GPU to speed computation too.
https://github.com/auduno/clmtrackr
